### PR TITLE
I've done my homework. Added: list, table, form.

### PR DESCRIPTION
**Pull request by oiskritsky**
I've done my homework.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,5 +13,117 @@ <h1>Моя первая домашняя страница</h1>
...
+                    <li>Теги для работы с таблицами.</li>
+                    <li>Теги для работы с формами.</li>
+                    <li>Div-блоки.</li>
+                </ul>
+            </li>
+        </ol>
+        <hr />
+        <table border="2" style="background-color:antiquewhite;  border-collapse: collapse;">
+            <th>Tag</th>
+            <th>Description</th>
```
Comment (ghost) [inline index.html:59]: Forgot to wrap `th`.

File: index.html [hunk 2]
```diff
@@ -13,5 +13,117 @@ <h1>Моя первая домашняя страница</h1>
...
+                <h3>What does HTTP stand for?</h3>
+                    <input id ="HTTP1" type="radio" name="http" value="true">
+                    <label for="HTTP1" >Hyper Text Translate Protocol</label><br />
+                    <input id ="HTTP2" type="radio" name="http" value=true">
+                    <label for="HTTP2" >Hyper Text Transmission Protocol</label><br />
+                    <input id ="HTTP3" type="radio" name="http" value="true">
+                    <label for="HTTP3" >Hyper Text Transfer Protocol</label><br />
+                <h3>Comment</h3>
+                    <textarea name="Comment" id="Comment" cols="30" rows="5" maxlength="500" placeholder="Comment"></textarea><br />
+                <h3>What does HTML stand for?</h3>
```
Comment (ghost) [inline index.html]: Why `h3`s doesn't have an indent.

File: index.html (hunk 1 above)
Comment (oiskritsky) [reply index.html:59]: wrap `th` in `tr`

File: index.html (hunk 2 above)
Comment (oiskritsky) [reply index.html]: fix it

**Pull request merged**